easybuild/tools/toolchain/toolchain.py

```diff
@@ -975,7 +975,8 @@ def is_rpath_wrapper(path):
         # need to use binary mode to read the file, since it may be an actual compiler command (which is a binary file)
         return b'rpath_args.py $CMD' in read_file(path, mode='rb')
 
-    def prepare_rpath_wrappers(self, rpath_filter_dirs=None, rpath_include_dirs=None):
+    def prepare_rpath_wrappers(self, rpath_filter_dirs=None, rpath_include_dirs=None, wrappers_dir=None,
+                               add_to_path=True, disable_wrapper_log=False):
```

Comment:
I need to rethink this slightly. I think a better approach is to have a build option `rpath_wrappers_dir` and drop these new arguments.